com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.cs

```diff
@@ -599,11 +599,17 @@ static void DrawLightIntensityGUILayout(SerializedHDLight serialized, Editor own
             Rect labelRect = lineRect;
             labelRect.width = EditorGUIUtility.labelWidth;
 
+            // Expand to reach both lines of the intensity field.
+            labelRect.height *= 2f;
```

Comment:
To be 100% accurate, the interline is missing here. You should have 2* singleLineHeight +2 (or 4 don't remember exactly and the property is not accessible in user land.)